CHANGELOG.md

```diff
@@ -1,4 +1,68 @@
+* **Datasource:** Fix dataproxy timeout should always be applied for outgoing data source HTTP requests. [#34597](https://github.com/grafana/grafana/pull/34597), [@dsotirakis](https://github.com/dsotirakis)
+* **Elasticsearch:** Fix NewClient not passing httpClientProvider to client impl. [#34539](https://github.com/grafana/grafana/pull/34539), [@KiVirgil](https://github.com/KiVirgil)
+* **Explore:** Fix Browser title not updated on Navigation to Explore. [#34651](https://github.com/grafana/grafana/pull/34651), [@axelavargas](https://github.com/axelavargas)
+* **GraphNG:** Remove fieldName and hideInLegend properties from UPlotSeriesBuilder. [#34901](https://github.com/grafana/grafana/pull/34901), [@dprokop](https://github.com/dprokop)
+* **OAuth:** Fix fallback to auto_assign_org_role setting for Azure AD OAuth when no role claims exists. [#34838](https://github.com/grafana/grafana/pull/34838), [@idafurjes](https://github.com/idafurjes)
+* **PanelChrome:** Fix issue with empty panel after adding a non data panel and coming back from panel edit. [#34765](https://github.com/grafana/grafana/pull/34765), [@torkelo](https://github.com/torkelo)
+* **StatPanel:** Fixes data link tooltip not showing for single value. [#34934](https://github.com/grafana/grafana/pull/34934), [@torkelo](https://github.com/torkelo)
+* **Table:** Fixes sorting for number fields. [#34722](https://github.com/grafana/grafana/pull/34722), [@hugohaggmark](https://github.com/hugohaggmark)
+* **Table:** datalink to have text underline and support for image datalink. [#34635](https://github.com/grafana/grafana/pull/34635), [@thisisobate](https://github.com/thisisobate)
+* **Time series panel:** Position tooltip correctly when window scrolled or resized. [#34584](https://github.com/grafana/grafana/pull/34584), [@dprokop](https://github.com/dprokop)
+* **Transformations:** Prevent FilterByValue transform crashing panel edit. [#34747](https://github.com/grafana/grafana/pull/34747), [@jackw](https://github.com/jackw)
+
+### Breaking changes
+
+
+The default HTTP method for Prometheus data source is now POST, previously it was GET. The POST APIs are there since January 2018 (Prometheus 2.1.0) and they have fewer limitations than the GET APIs (e.g. when dealing with high cardinality labels, GET hits the URL size limit). Users with Prometheus instance with version < 2.1.0 that use the default HTTP method should update their HTTP method to GET. Issue [#34599](https://github.com/grafana/grafana/issues/34599)
+
+### Deprecations
+
+
+![image](https://user-images.githubusercontent.com/46142/119644192-8383ac00-be14-11eb-9b17-6eb0217f958c.png)
+![image](https://user-images.githubusercontent.com/46142/119644260-96967c00-be14-11eb-8489-a82ef2c2c443.png)
+
+
+**Which issue(s) this PR fixes**:
+
+Fixes #33709
+ Issue [#34725](https://github.com/grafana/grafana/issues/34725)
```

Comment:
hmm, this looks wrong?

Reply:
Yeah

Reply:
@joshhunt can you check why https://github.com/grafana/grafana/pull/34725 is ending up in the deprecations part of the release notes. Maybe try not to use the word deprecation? I'm not sure.

Reply:
🤦🏻 The inline text "Deprecation notices" was enough to trigger this getting added here.

I've updated the PR description - should this re-run and it'll regenerate?

Reply:
I'm just guessing, but I think that the next time we run the update change log action your changes to the description will show @joshhunt. Thank you for helping out!